pom.xml

```diff
@@ -1671,6 +1668,57 @@
         <module>yarn</module>
         <module>network/yarn</module>
       </modules>
+      <properties>
+        <jersey.version>1.9</jersey.version>
+      </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
```

Comment:
This comment (and the `<activation>` part) should not be added here. The `yarn` profile is never automatically enabled, and we need to keep it that way.